modules/foreman_installation_medium.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2018 Manuel Bonk (ATIX AG)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: foreman_installation_medium
+short_description: Manage Foreman Installation Medium using Foreman API
+description:
+  - Create and Delete Foreman Installation Medium using Foreman API
+version_added: "2.5"
+author:
+  - "Manuel Bonk(@manuelbonk) ATIX AG"
+requirements:
+  - nailgun >= 0.16.0
+options:
+  name:
+    description: The full installation medium name
+    required: true
+  locations:
+    description: List of locations the installation medium should be assigned to
+    required: false
+    type: list
+  operatingsystems:
+    description: List of operating systems the installation medium should be assigned to
```

Comment:
Can you make it clear it should be the title? People could think name might work.

Reply:
If it were unique, the name would actually work, as it is part of the title.